x-pack/plugins/security_solution/public/resolver/store/data/graphing.test.ts

```diff
@@ -187,3 +204,48 @@ describe('resolver graph layout', () => {
     });
   });
 });
+
+describe('resolver graph with too much lineage', () => {
+  let generator: EndpointDocGenerator;
+  let store: Store<DataState, DataAction>;
+  let allEvents: ResolverEvent[];
+  let childrenCursor: string;
+  let ancestorCursor: string;
+
+  beforeEach(() => {
+    generator = new EndpointDocGenerator('seed');
+    allEvents = generator.generateTree({ ancestors: 1, generations: 2, children: 2 }).allEvents;
+    childrenCursor = 'aValidChildursor';
```

Comment:
missing a 'c' in cursor

Reply:
https://github.com/elastic/kibana/pull/70097/commits/bce11223b6875da6a2ec0d9279ca73eb8b607026